public/js/src/bridge/bridge_lb_status.tag

```diff
@@ -0,0 +1,69 @@
+<bridge_lb_status>
```

Comment:
so what happens if I navigate away from this shipment while the interval is still going on? It looks like this logic will still be in effect, even though the user is not on the page. I think we need to add something else in, that stops the interval once the page is navigated away from.